Plugins/Public/autobuy/Main.cpp

```diff
@@ -699,12 +763,38 @@ void PlayerAutobuy(uint iClientID, uint iBaseID)
 			}
 
 			//FLHook handling
-			if (mapAutobuyFLHookExtras.find(eq->iArchID) != mapAutobuyFLHookExtras.end())
+			if (mapAutobuyFLHookCloak.find(eq->iArchID) != mapAutobuyFLHookCloak.end())
```

Comment:
I believe this is an opportunity to stop doing these unnecessary nested if statements and just use &&

Reply:
Fixed